src/python/director/startup.py

```diff
@@ -685,38 +686,45 @@ def onPlanStatus(msg):
         if polyData:
             vis.showPolyData(polyData, os.path.basename(filename))
 
-
-if useImageWidget:
-    imageWidget = cameraview.ImageWidget(cameraview.imageManager, 'CAMERA_LEFT', view)
-    #imageWidget = cameraview.ImageWidget(cameraview.imageManager, 'KINECT_RGB', view)
-
 if useCameraFrustumVisualizer and cameraview.CameraFrustumVisualizer.isCompatibleWithConfig():
     cameraFrustumVisualizer = cameraview.CameraFrustumVisualizer(robotStateModel, cameraview.imageManager, 'CAMERA_LEFT')
 
 class ImageOverlayManager(object):
 
     def __init__(self):
         self.viewName = 'CAMERA_LEFT'
-        #self.viewName = 'KINECT_RGB'
-        self.size = 400
+        self.desiredWidth = 400
+        self.size = [self.desiredWidth, self.desiredWidth/(4/3.)]
```

Comment:
this should be reverted, the aspect ration needs to be defined by the image dimensions after the first image is received by lcm, it should not be hard coded